### ISSUE TITLE
godot: "noinline" option is not present in json schema

### ISSUE THREAD

**Issue by ppipada** [golangci/golangci-lint#5921]
### Welcome

- [x] Yes, I'm using a binary release within 2 latest [releases](https://github.com/golangci/golangci-lint/releases). Only such installations are supported.
- [x] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [x] Yes, I've read the `typecheck` section of the [FAQ](https://golangci-lint.run/welcome/faq/#why-do-you-have-typecheck-errors).
- [x] Yes, I've tried with the standalone [linter](https://golangci-lint.run/usage/linters/) if available (e.g., gocritic, go vet, etc.).
- [x] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### How did you install golangci-lint?

go install

### Description of the problem

godot: "noinline" option is not present in json schema of godot settings and hence vscode fails to recognize and validate that option setting in yml file. 

The PR where godots 1.5.0 was merged: https://github.com/golangci/golangci-lint/pull/5420

Crosschecked schema files in toplevel jsonschema folder.

### Version of golangci-lint

<details>

```console
golangci-lint has version 2.1.6 built with go1.24.2 from eabc2638 on 2025-05-04T15:41:19Z
```

</details>

### Configuration

<details>

```console

linters:
  settings:
    godot:
      # Comments to be checked: `declarations`, `toplevel`, `noinline` or `all`.
      # Default: declarations
      # This is shown as error in jsonschema validated against:  https://golangci-lint.run/jsonschema/golangci.jsonschema.json
      scope: noinline
      capital: true
# paste configuration file or CLI flags here
```
</details>

### Go environment

<details>

```console
go version go1.24.2 linux/amd64
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# paste output here
```

</details>

### A minimal reproducible example or link to a public repository

<details>

```go
NA
```

</details>

### Validation

- [x] Yes, I've included all information above (version, config, etc.).

### Supporter

- [ ] I am a sponsor/backer through [GitHub](https://github.com/sponsors/golangci/) or [OpenCollective](https://opencollective.com/golangci-lint)

**Comment by ldez** [comment]
`noinline` is not a dedicated option but a value of `scope`.

I will update the JSON schema.

**Comment by ppipada** [comment]
Thank you !!!
